research/format.markdown

```diff
@@ -0,0 +1,97 @@
+# Research on `format` keyword 
+Changes in `format` Keyword Behaviour Across all JSON Schema Drafts: Additions, Deletions, and Upgrades.
+
+## Draft 0 to Draft 2 (No changes)
+
+## Draft 2 to Draft 3
+
+References: [Draft 2 Section 5.20](https://json-schema.org/draft-02/draft-zyp-json-schema-02.txt), [Draft 3 Section 5.23](https://json-schema.org/draft-03/draft-zyp-json-schema-03.pdf)
+
+> Behaviour:   
```

Comment:
This behavior change seems to be exactly the same. In both cases its a "maybe"?

Reply:
That means in Draft 2 by default it was annotation. not exactly annotation but  should not validate by default.
But in Draft 3 its implementors choice

Reply:
This last explanation you just gave me is clearer. Maybe clarify on the document as the current explanation is ambiguous?

Reply:
Yeah I will rite more to clarify. I had copy pasted the exact lines from draft, which I thought was best to do